templates/vue/src/fixtures/deep-page.json

```diff
@@ -55,7 +55,7 @@
       "hideProgress": false,
       "hideMedia": false,
       "skippable": true,
-      "quiz": [],
+      
```

Comment:
Again, these fixtures have unnecessarily added empty lines. Please remove here and the other files.